libs/damap/src/lib/widgets/flip-card/flip-card.component.css

```diff
@@ -0,0 +1,100 @@
+.flip-card {
+  background-color: transparent;
+  width: 350px;
+  height: 150px;
+  perspective: 1000px;
+  border-radius: 12px !important;
+  cursor: default !;
+}
+
+.flip-card-inner {
+  position: relative;
+  width: 100%;
+  height: 100%;
+  text-align: center;
+  transition: transform 0.6s;
+  transform-style: preserve-3d;
+  box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2);
+  border-radius: 12px !important;
+}
+
+.flip-card:hover .flip-card-inner {
+  transform: rotateY(180deg);
+}
+
+.flip-card:focus {
+  outline: 3px solid var(--accent-color-lighter);
+}
+
+.flip-card-front,
+.flip-card-back {
+  position: absolute;
+  width: 100%;
+  height: 100%;
+  -webkit-backface-visibility: hidden !important;
+  backface-visibility: hidden !important;
+  border-radius: 12px !important;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.flip-card-front {
+  background-color: var(--primary);
+  color: var(--primary-lightest);
```

Comment:
This should be `color: var(--on-primary);`. On means that this is the contrast color used when something has primary as background.